FundTrack.BLL/Concrete/FinOpService.cs

```diff
@@ -157,64 +157,142 @@ public FinOpViewModel CreateTransfer(FinOpViewModel finOpModel)
             FinOpInputDataValidation(finOpModel);
             try
             {
-                var orgAccFrom = _unitOfWork.OrganizationAccountRepository.GetOrgAccountById(finOpModel.CardFromId);
-                var orgAccTo = _unitOfWork.OrganizationAccountRepository.GetOrgAccountById(finOpModel.CardToId);
-                if (finOpModel.Sum > orgAccFrom.CurrentBalance)
+                var orgAccFrom = unitOfWork.OrganizationAccountRepository.GetOrgAccountById(finOpModel.CardFromId);
+                var orgAccTo = unitOfWork.OrganizationAccountRepository.GetOrgAccountById(finOpModel.CardToId);
+                if (finOpModel.Amount > orgAccFrom.CurrentBalance)
                 {
                     throw new ArgumentException("Витрати не можуть перебільшувати баланс рахунку");
                 }
                 var finOp = new FinOp
                 {
-                    Amount = finOpModel.Sum,
+                    Amount = finOpModel.Amount,
                     AccToId = orgAccTo.Id,
                     AccFromId = orgAccFrom.Id,
                     Description = finOpModel.Description,
                     FinOpDate = finOpModel.Date,
                     FinOpType = finOpModel.FinOpType,
                     UserId = finOpModel.UserId
                 };
-                _unitOfWork.FinOpRepository.Create(finOp);
-                orgAccFrom.CurrentBalance -= finOpModel.Sum;
-                _unitOfWork.OrganizationAccountRepository.Edit(orgAccFrom);
-                orgAccTo.CurrentBalance += finOpModel.Sum;
-                _unitOfWork.OrganizationAccountRepository.Edit(orgAccTo);
-                _unitOfWork.SaveChanges();
+                unitOfWork.FinOpRepository.Create(finOp);
+                orgAccFrom.CurrentBalance -= finOpModel.Amount;
+                unitOfWork.OrganizationAccountRepository.Edit(orgAccFrom);
+                orgAccTo.CurrentBalance += finOpModel.Amount;
+                unitOfWork.OrganizationAccountRepository.Edit(orgAccTo);
+                unitOfWork.SaveChanges();
                 return finOpModel;
             }
             catch (Exception ex)
             {
                 throw new BusinessLogicException("Щось пішло не так....О_о", ex);

```

Comment:
Since you have edited an operation in try block, you should know what may go wrong. Please edit the ddecription in catch block